node/src/reactor.rs

```diff
@@ -60,6 +61,9 @@ use crate::{
 use quanta::Clock;
 pub use queue_kind::QueueKind;
 
+/// Default threshold for amount of total ram remaining before dumping queues to disk.
+const MEM_DUMP_THRESHOLD: f32 = 0.25;
```

Comment:
Would it be useful to have this as a config value?  I can see a need to tweak this value somewhat as we get more experience.

Reply:
I'm not clear on how I can access the R::Config value from this point of view - perhaps I can work on this in a follow-up PR. I also tried plumbing through an environment variable, similar to `DISPATCH_EVENT_THRESHOLD`  but was stuck on getting nctl to pick up on it.

Reply:
>  was stuck on getting nctl to pick up on it.

Instead of exporting the value, try supplying it in the same line as the nctl command, like this `MEM_DUMP_THRESHOLD=0.1 nctl-start node=6`

Reply:
Right - I see the issue now.  I'd be fine with making it an env var for now, and we can look to make it a config value proper in a future PR if required.

Reply:
I actually think it's better as an env var. If we put it in the chainspec then it will appear in validators' configs and they will start asking, tweaking etc. This is for debugging purposes mostly. Let's make it an env var and default to some reasonable value.